src/solidity-frontend/solidity_convert.cpp

```diff
@@ -199,6 +202,42 @@ bool solidity_convertert::convert()
   return false; // 'false' indicates successful completion.
 }
 
+void solidity_convertert::multi_Json_file(nlohmann::json &src_ast_json)
```

Comment:
Let's stick to lower-case symbol names delimited by underscores.

Reply:
I'll correct it immediately.